app/src/main/AndroidManifest.xml

```diff
@@ -116,6 +126,9 @@
             </intent-filter>
         </service>
 
+        <meta-data
+            android:name="preloaded_fonts"
```

Comment:
For the search widget text, when I set the font to Roboto, the `preloaded_fonts` file was automatically created. Is there already a place to consolidate all the fonts?

Reply:
For the voice icon, does your phone have text-to-speech available? I think I have to add a check for that in that case.

Reply:
For the preview image, I think I need an actual image of the search widget and place it in the drawable folder. Should I ask someone from the UX team to do that?

Reply:
We currently try to use system fonts instead of including custom fonts. Would it be possible to use the system sans-serif font instead? (It's usually Roboto anyways)

Reply:
Yeah, I didn't know that it uses Roboto as default. I'll change that!

Reply:
Sounds good to me :)